packages/components/button/src/button.ts

```diff
@@ -37,8 +41,19 @@ export const buttonProps = buildProps({
   autofocus: Boolean,
   round: Boolean,
   circle: Boolean,
+  autoInsertSpace: {
+    type: Boolean,
+  },
 } as const)
 
+export interface ButtonGlobalConfig {
+  autoInsertSpace: boolean
+}
```

Comment:
We just make properties optional
```suggestion
export interface ButtonGlobalConfig {
  autoInsertSpace?: boolean
}
```
And use optional chaining

Reply:
Actually it was not called `optional chaining` here, it's just represented for `optional` BTW. :0